portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,51 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>My Portfolio - Jose Ramirez</title>
     <link rel="stylesheet" href="style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;0,800;1,300;1,400;1,600;1,700;1,800&display=swap"
+      rel="stylesheet">
```

Comment:
Consider testing in mobile view as well

![image](https://user-images.githubusercontent.com/5449100/83543463-6d296580-a4ca-11ea-97f9-19a57e6feb09.png)

Right now it seems like the titles are a little bit quirky on mobile.

Something that could help is the meta tag for the viewport, so that your site maps correctly when viewed in a mobile device.

https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag
https://stackoverflow.com/a/31680188

Reply:
Fixed in latest commit

Reply:
Could you fix the size of the subtitles under "Education" and "Work Experience" please? they are looking a bit too small in mobile

Reply:
This has already been fixed! The preview was outdated